q2_chemistree/tests/test_match_table.py

```diff
@@ -0,0 +1,47 @@
+import functools
+from unittest import TestCase, main
+from biom.table import Table
+import pandas as pd
+import os
+from q2_chemistree import match_table, collatefp, make_hierarchy
+
+
+class test_match(TestCase):
+    def setUp(self):
+        THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+        table = Table({}, [], [])
```

Comment:
Is this needed?

Reply:
I _think_ so? We're testing for the situation when user inputs an empty feature table

Reply:
I guess I'm confused as the `match_table` method says in the docstring that it takes a `pd.DataFrame` but this is a `biom.Table`. Is this actually used, and is it safe to provide a `biom.Table` to `match_table`?

Reply:
you're right, good catch, thanks - fixing.